### PR TITLE
Initial support for ujson (as a Python fuzzing sample).

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
This adds the fuzzers and builds ujson in the correct format for Atheris to use it. It doesn't yet copy the fuzzers to the archive (pending whatever we decide for how we want to bundle things together). I'm going to hold off a little bit on reaching out to the maintainers, but it looks like the project is well-maintained and we should be able to add a reasonable primary contact when we're ready.

### CODE REVIEW

File: projects/ujson/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build and install fuzzers
+# TODO(mbarbella): Don't set CFLAGS/CXXFLAGS directly.
```
Comment (mbarbella-chromium) [inline projects/ujson/build.sh:19]: Do we have a clean way to build with fuzzer-no-link?

**Review by inferno-chromium — APPROVED** [review]

**Review by inferno-chromium — APPROVED** [review]
LGTM

**Review by oliverchang — COMMENTED** [review]
do we need any changes on CF to support this? 

If we do, we might need to figure out the story on how to package these targets first.

**Comment by mbarbella-chromium** [comment]
Yeah, this is going to need some CF-side changes.

**Comment by inferno-chromium** [comment]
Will merge now, we need more stuff to fix runtime, hopefully we land today as well.

**Pull request merged**